package/yast2-storage-ng.spec

```diff
@@ -15,9 +15,9 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
-Name:		yast2-storage-ng
-Version:	4.1.29
-Release:	0
+Name:    yast2-storage-ng
+Version: 4.1.30
+Release: 0
```

Comment:
Why those changes in spacing?

Reply:
I don't know, I only run a `rake version:bump`. Let me fix it.